### ISSUE TITLE
the minified version will create a global variable `d`

### ISSUE THREAD

**Issue by lpetrov** [kayahr/jquery-fullscreen-plugin#21]
In most cases this should not be a problem, but creating global variables is not a good idea.

The minified version should be scoped by a (function(){ ... })() and only expose the expected/required/public functions/variables, e.g. $.fullScreen.

**Comment by humancalendar** [comment]
Yeah, it looks like the supplied minified version (jquery.fullscreen-min.js) has not been updated since its code doesn't match that of the non-minified version.  The non-minified version has the correct scoping as you indicate, but the supplied minified version does not.

I manually minified the jquery.fullscreen.js file and ended up with the proper scoping.
